web/packages/teleport/src/services/agents/types.ts

```diff
@@ -23,8 +23,8 @@ import { Database } from 'teleport/services/databases';
 import { Node } from 'teleport/services/nodes';
 import { Kube } from 'teleport/services/kube';
 import { Desktop } from 'teleport/services/desktops';
-
-import { UserGroup } from '../userGroups';
+import { UserGroup } from 'teleport/services/userGroups';
+import { GitServer } from 'teleport/services/gitservers';
```

Comment:
I don't have any strong opinion, but `gitservers` seems inconsistent with other exports. I'd rename it to `gitServers`.